sbpl_lattice_planner/src/sbpl_lattice_planner.cpp

```diff
@@ -206,6 +210,7 @@ void SBPLLatticePlanner::initialize(std::string name, costmap_2d::Costmap2DROS*
     ROS_INFO("[sbpl_lattice_planner] Initialized successfully");
     plan_pub_ = private_nh.advertise<nav_msgs::Path>("plan", 1);
     stats_publisher_ = private_nh.advertise<sbpl_lattice_planner::SBPLLatticePlannerStats>("sbpl_lattice_planner_stats", 1);
+    sbpl_plan_footprint_pub_ = private_nh.advertise<visualization_msgs::Marker>("sbpl_plan_footprint_array", 1);
```

Comment:
Maybe a shorter name, like "footprint_markers", would be better? It's in the private namespace anyway, so there won't be any name collisions. Also, it's not an array.